File: 268-missing-number/268-missing-number.cs
public class Solution {
    public int MissingNumber(int[] nums) {
      int sum = 0;
      foreach(var num in nums) {
          sum += num;
      }
      int n = nums.Length;
      int target = n * (n + 1) / 2;
      return target - sum;
    }
}

File: 268-missing-number/268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n = nums.size();
        long sum = 0;
        for(int item : nums)sum += item;
        long long multiplication = 1LL * n * (n + 1) / 2;
        return int(multiplication - sum);
    }
};